.github/workflows/daily-build.yml

```diff
@@ -0,0 +1,38 @@
+name: Daily build
+
+on:
+  schedule:
+    - cron:  '0 */12 * * *'
```

Comment:
This means that the workflow will run every 12 hours?

Reply:
Yes.

Reply:
Isn't it should be every 24 hours?